### DISCUSSION TITLE
Duplication glitch with Integrated Terminals (+ suggestions for tweaks)

### DISCUSSION THREAD

**Discussion by l4ssc** [ACGaming/UniversalTweaks#463] [Bugfix Suggestions]
[demonstration](https://streamable.com/zh9thx)

Replicates in an instance with just Integrated Terminals. Integrated Dynamics' mods are not maintained anymore for 1.12 so outside fixes for this are necessary.

(Bonus points if anything can be done about IT's incompatibility with Inventory Tweaks or Mouse Tweaks or its weird UX with it trying to extract as many of an item as possible into the player's inventory on shift-click)

**Comment by jchung01** [comment]
You can check out this [fix mod](https://www.curseforge.com/minecraft/mc-mods/integrated-derivative) for Integrated Dynamics related fixes and improvements, including the ones you mentioned.
